dhis-2/pom.xml

```diff
@@ -1,1405 +1,1411 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>org.hisp.dhis</groupId>
-  <artifactId>dhis</artifactId>
-  <version>2.32-SNAPSHOT</version>

```

Comment:
Can we avoid reformatting this file

Reply:
Yes, I found a way to fix this issue. Hopefully, this is the last time I reformat this file. It has to do with line ending ON THIS PARTICULAR file (other pom files are not affected). Sorry about that